### PR TITLE
252 change facets requests

### PR DESCRIPTION
**Pull request by kamerynB**

### CODE REVIEW

File: client/src/lib/util/timelineHelper.ts [hunk 1]
```diff
@@ -1,19 +1,170 @@
...
+
+/**
+ * Adds the correct year label to the string year
+ * @param {string} year; the year to add the label
+ * @returns {string}
+ */
+export const getYearWithLabel = (year: string): string =>
+  year.includes('-') ? `${year.substring(1)} B.C.E.` : `${year} C.E.`
+
+/**
```
Comment (clarkepeterf) [inline client/src/lib/util/timelineHelper.ts:111]: Can you remove all of the code commented out between lines 111 and 170?

File: client/src/lib/util/fetchTimeline.ts [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const getTimelines = (
+  hrefs: Array<string>,
+): Promise<{ data: (Error | { [x: string]: any })[] }> => {
+  const promises = hrefs.map((href: string) => fetchTimelineData(href))
+  return Promise.all(promises).then((result) => ({ data: result }))
+}
+
+// // eslint-disable-next-line @typescript-eslint/no-explicit-any
```
Comment (clarkepeterf) [inline client/src/lib/util/fetchTimeline.ts:23]: Can you remove all of the code commented out between line 23 and 68?

File: client/src/features/timeline/TimelineContainer.tsx [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+            </Col> */}
+          </Row>
+          <Row>
+            <Col xs={12}>
+              <List
+                sortedKeys={sortedKeys}
+                transformedData={transformedData}
+                searchTags={searchTags}
+              />
+              {/* {display === 'list' ? (
```
Comment (clarkepeterf) [inline client/src/features/timeline/TimelineContainer.tsx:121]: can you remove the code commented out between lines 121 and 128?

File: client/src/features/timeline/TimelineContainer.tsx [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+      return (
+        <StyledEntityPageSection
+          data-testid="timeline-container"
+          ref={timelineRef}
+        >
+          <Row>
+            <Col xs={8}>
+              <h2>Timeline of Related Objects/Works</h2>
+            </Col>
+            {/* <Col xs={4} className="d-flex justify-content-end">
```
Comment (clarkepeterf) [inline client/src/features/timeline/TimelineContainer.tsx:75]: can you remove the code commented out between lines 75 and 112?

File: client/src/features/timeline/TimelineContainer.tsx [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+  providedHalLinks: any
+}> = ({ searchTags, providedHalLinks }) => {
+  const links = getHalLinks(searchTags, providedHalLinks)
+  // const [isFullscreen, setIsFullscreen] = useState<boolean>(false)
+  const timelineRef = useRef<HTMLDivElement>(null)
+  // const [display] = useState<'list' | 'graph'>('list')
+
+  const { data, isSuccess, isError } = useGetTimelineQuery(links)
+
+  // const setFullscreen = (): void => {
```
Comment (clarkepeterf) [inline client/src/features/timeline/TimelineContainer.tsx:49]: can you remove the code commented out between lines 49 and 59?

Comment (kamerynB) [reply client/src/features/timeline/TimelineContainer.tsx]: Removed.

File: client/src/features/timeline/TimelineContainer.tsx (hunk 4 above)
Comment (kamerynB) [reply client/src/features/timeline/TimelineContainer.tsx]: Removed

File: client/src/features/timeline/TimelineContainer.tsx (hunk 3 above)
Comment (kamerynB) [reply client/src/features/timeline/TimelineContainer.tsx]: Removed.

File: client/src/lib/util/fetchTimeline.ts (hunk 2 above)
Comment (kamerynB) [reply client/src/lib/util/fetchTimeline.ts]: Removed.

File: client/src/lib/util/timelineHelper.ts (hunk 1 above)
Comment (kamerynB) [reply client/src/lib/util/timelineHelper.ts:111]: Removed.

**Pull request merged**